conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -11,3 +11,6 @@
 
 # Add all the application routes to the prod.routes file
 ->         /                          prod.Routes
+
+GET        /test-only/claim            uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.PostNewClaimController.claim
```

Comment:
Also, this will trash your generated routes in the base controller package. This is why you were seeing compilation errors yesterday with respect to using the reverse routes. This will trash the base controller routes. You either have to not use the testOnly routes and have that controller in the main controller package OR you take that controller out of the main controller package and put it in a separate package for testing ie testOnly package. That is the normal way to do this.